include/dlaf/eigensolver/internal/get_red2band_panel_nworkers.h

```diff
@@ -19,6 +19,10 @@
 
 namespace dlaf::eigensolver::internal {
 
+inline size_t get_red2band_panel_worker_minwork() noexcept {
+  return 1;
```

Comment:
It seems to me it'd probably be worth it to make this a configuration option, the cost is not so high, but I won't block this PR on that.

Reply:
fully agree. if also @rasolca I would go for adding it.